templates/page.values.php

```diff
@@ -49,6 +49,6 @@
 			</li>
 		</ul>
 
-		<p class="details-link"><a href="<?php p($theme->getBaseUrl()); ?>" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', $theme->getName())); ?></a></p>
+		<p class="details-link"><a href="https://nextcloud.com" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', [ \OC_Defaults::getProductName() ])); ?></a></p>
```

Comment:
```suggestion
		<p class="details-link"><a href="https://nextcloud.com" target="_blank" rel="noreferrer noopener"><?php p($l->t('Learn more about %s', [ $theme->getProductName() ])); ?></a></p>
```

Otherwise fine for now :+1: 

For the proper theming of the url we'd need something similar to https://github.com/nextcloud/server/pull/25950 + https://github.com/nextcloud/server/pull/27529 for the url in addition to the product name

Reply:
done